apps/etterlatte-beregning/src/main/kotlin/avkorting/AvkortingService.kt

```diff
@@ -86,6 +89,34 @@ class AvkortingService(
         }
     }
 
+    // Sjekke om Avkorting innvilga måneder overstyres på grunn av tidlig alderspensjon, og opprett oppgave om opphør
+    suspend fun opprettOppgaveHvisTidligAlderspensjon(
+        behandlingId: UUID,
+        brukerTokenInfo: BrukerTokenInfo,
+    ) {
+        val avkorting = avkortingRepository.hentAvkorting(behandlingId) ?: Avkorting()
+        val behandling = behandlingKlient.hentBehandling(behandlingId, brukerTokenInfo)
+
+        val aarsoppgjoer = avkorting.aarsoppgjoer.single { it.aar == behandling.virkningstidspunkt?.dato?.year }
+        val overstyrtInntektsavkorting =
+            aarsoppgjoer.inntektsavkorting
+                .find {
+                    it.grunnlag.overstyrtInnvilgaMaanederAarsak == OverstyrtInnvilgaMaanederAarsak.TAR_UT_PENSJON_TIDLIG &&
+                        it.grunnlag.periode.fom == behandling.virkningstidspunkt?.dato
+                }
+
+        if (overstyrtInntektsavkorting != null) {
```

Comment:
Burde man logge noe i tilfeller hvor denne er `null`?

Reply:
Ja, kan være greit for å vite om vi har tatt stilling til saken.